workflow/Activity/getItemsdetailActivity.go

```diff
@@ -0,0 +1,17 @@
+package activity
+
+import (
+	"log"
+
+	model "github.com/E-Furqan/Food-Delivery-System/Models"
+)
+
+func (act *Activity) GetItems(order model.CombineOrderItem, token string) ([]model.Items, error) {
+
+	items, err := act.ResClient.GetItems(order)
+	if err != nil {
+		log.Print("error from get items:", err)
+		return []model.Items{}, err
+	}
+	return items, nil
+}
```

Comment:
filename should be itemsactivity, we dont need to create separate file for each endpoint

Reply:
have changed the file to restaurant activity so that in future if we have more activities related to restaurant.